go/chat/tlf.go

```diff
@@ -107,12 +108,9 @@ func (t *KBFSNameInfoSource) CryptKeys(ctx context.Context, tlfName string) (res
 	if runIdentify {
 		t.Debug(ectx, "CryptKeys: running identify")
 		group.Go(func() error {
-			query := keybase1.TLFQuery{
-				TlfName:          tlfName,
-				IdentifyBehavior: identBehavior,
-			}
 			var err error
-			ib, err = t.Identify(ectx, query, true)
+			names := strings.Split(strings.Fields(tlfName)[0], ",")
```

Comment:
This little guy could be a function.  It's repeated several times.  In convsource.go too.